docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:16.04
+MAINTAINER koalaman <koalaman@gmail.com>
+
+RUN apt-get update -qy
+
+RUN apt-get -qy install git cabal-install
+
+RUN cd $HOME && git clone https://github.com/koalaman/shellcheck.git
```

Comment:
This is going to result in a cached layer, so you will never get an updated version of the repository unless you invalidate the intermediate layer